packages/verified-fetch/src/index.ts

```diff
@@ -784,34 +792,75 @@ export interface VerifiedFetchInit extends RequestInit, ProgressOptions<BubbledP
   allowInsecure?: boolean
 }
 
+type Libp2pServiceType = Pick<DefaultLibp2pServices, 'dcutr' | 'identify' | 'keychain' | 'ping' > & Record<`delegatedRouting${number}`, any>
```

Comment:
small nit. type is exported https://github.com/ipfs/helia-delegated-routing-v1-http-api/blob/1e60306ee391e9400dbc098fee5ebd4eb0bf7562/packages/client/src/index.ts#L162C18-L162C49 
```suggestion
type Libp2pServiceType = Pick<DefaultLibp2pServices, 'dcutr' | 'identify' | 'keychain' | 'ping' > & Record<`delegatedRouting${number}`, DelegatedRoutingV1HttpApiClient>
```